lib/browser/api/web-contents.ts

```diff
@@ -461,8 +461,11 @@ const addReturnValueToEvent = (event: any) => {
   });
 };
 
+const { hasSwitch } = process._linkedBinding('electron_common_command_line');
+const { hasVar } = process._linkedBinding('electron_common_environment');
```

Comment:
nit:
```suggestion
const commandLine = process._linkedBinding('electron_common_command_line');
const environment = process._linkedBinding('electron_common_environment');
```

I think these read better if not destructured (e.g. `commandLine.hasSwitch()` and `environment.hasVar()`)